examples/bno055_i2c-gpio_simpletest.py

```diff
@@ -0,0 +1,30 @@
+"""
+This example demonstrates how to instantiate the
+Adafruit BNO055 Sensor using this library and just
+the I2C bus number.
+This example will only work on a Raspberry Pi
+and does require the i2c-gpio kernel module to be
+installed and enabled. Most Raspberry Pis will
+already have it installed.
+"""
+
+import time
+from adafruit_extended_bus import ExtendedI2C as I2C
+import adafruit_bno055
+
+# Create library object using our Extended Bus I2C port
```

Comment:
Add comment here showing the line you used in config.txt to enable and configure the i2c-gpio overlay to go along with this example.